modules/space/rendering/renderablesmallbody.cpp

```diff
@@ -67,131 +66,44 @@ namespace {
         "method includes lines. If the rendering mode is set to Points, this value is "
         "ignored."
     };
-    constexpr openspace::properties::Property::PropertyInfo FadeInfo = {
-        "Fade",
-        "Line fade",
-        "The fading factor that is applied to the trail if the 'EnableFade' value is "
-        "'true'. If it is 'false', this setting has no effect. The higher the number, "
-        "the less fading is applied."
-    };
     constexpr openspace::properties::Property::PropertyInfo LineColorInfo = {
         "Color",
         "Color",
         "This value determines the RGB main color for the lines and points of the trail."
     };
-    
-    constexpr const char* KeyFile = "Path";
-    constexpr const char* KeyLineNum = "LineNumber";
+    constexpr openspace::properties::Property::PropertyInfo TrailFadeInfo = {
```

Comment:
same q as above

Reply:
See related answer in renderablesatellites.cpp